sw/device/lib/ujson/ujson_ottf.h

```diff
@@ -0,0 +1,55 @@
+// Copyright lowRISC contributors.
```

Comment:
nit/question: since these are specific implementations related to the OTTF, should these be placed in the `.../test_framework/` dir?

Reply:
Done.